subgraph/config/mainnet.json

```diff
@@ -8,6 +8,10 @@
     "address": "0x7e184B0cC12572D12Db6dA248322A0e3618fc756",
     "startBlock": 20235530
   },
+  "withdrawalQueue": {
+    "address": "0x0",
```

Comment:
Artifact available in https://github.com/acre-btc/acre/pull/991

Reply:
8ad5841